src/test/java/org/blyznytsia/context/data/Bean7.java

```diff
@@ -0,0 +1,14 @@
+package org.blyznytsia.context.data;
+
+import lombok.Data;
+import org.blyznytsia.annotation.Autowired;
+import org.blyznytsia.annotation.Component;
+
+import java.util.List;
+
+@Component
+@Data
+public class Bean7 {
+  @Autowired
+  List<Bean3> bean3List;
```

Comment:
a proper example should be a list parametrized with an interface type 
e.g.
``` java
  @Autowired List<Convertor> convertors;
```
where `Convertor` is of the interface type.